### PR TITLE
Jkw sleep/hydration tests

### PR DESCRIPTION
**Pull request by jkwest-93**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features
- [x] Testing

### Detailed Description
- Adds two new testing suites: one for `Sleep` class and the other for `Hydration` data classes
- Writes tests each class
- Updates `Hydration Repo` tests to include new methods

### Why is this change required? What problem does it solve?
- After refactoring to have a `Data Repo`, & extending new methods, we had to update our tests to reflect our methods are working.

### Were there any challenges that arose while implementing this feature? If so, how were the addressed?
-n/a

### Files modified:
- [ ] index.html
- [ ] style.scss
- [ ] scripts.js
- [ ] Activity.js
- [ ] Hydration.js
- [ ] Sleep.js
- [ ] User.js
- [x] Other files: Hydration-test.js, Hydration-repo-test.js, Sleep-test.js

### CODE REVIEW

**Comment by krogowsk531** [comment]
Great job working with this test file!

File: test/hydration-repo-test.js [hunk 1]
```diff
@@ -65,20 +95,23 @@ describe('Hydration', function() {
   });
```
Comment (rwilliams659) [inline test/hydration-repo-test.js:95]: Awesome job, just a few suggested additional tests: add a test for if hydration is an instance of HydrationRepo & that it's an instance of DataRepo

File: test/hydration-test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      "date": "2019/06/15",
+      "numOunces": 37
+    });
+  });
+
+    it('should be a function', function() {
+      expect(Hydration).to.be.a('function');
+    });
+
+    it('should be an instance of Sleep', function() {
```
Comment (rwilliams659) [inline test/hydration-test.js:20]: it statement should say 'should be an instance of Hydration'!

Comment (jkwest-93) [reply test/hydration-test.js]: Great catch! Updated this.

File: test/hydration-repo-test.js (hunk 1 above)
Comment (jkwest-93) [reply test/hydration-repo-test.js:108]: Thanks for that- I totally leaned into the old testing suite too hard & didn't see these gaps.

**Pull request merged**